tutorial/documents.md

```diff
@@ -46,8 +45,9 @@ Models are generally differentiated by the contents options used to fetch the mo
 
 ### [Widget Extensions](http://jupyter.org/jupyterlab/classes/_docregistry_registry_.documentregistry.html#addwidgetextension)
```

Comment:
This feels out of place. The above text introduces the concept of the Document Manager, and the below text describes the Document Manager, but this is not immediately related to either. Perhaps it can be moved further down?